apps/frontend/src/components/form/MedicalForm.tsx

```diff
@@ -0,0 +1,275 @@
+} from '@chakra-ui/react';
+import { PhoneIcon } from '@chakra-ui/icons';
+import { Field, FieldProps } from 'formik';
+
+enum CancersDropdownValues {
+  BRAINCANCERS = 'braincancers',
+  LEUKEMIAS='leukemias',
+  LYMPHOMAS='lymphomas',
+  THYROIDCANCER='thyroidCancer',
+  GERMCELLTUMORS='germCellGonadalTumors',
+  NEUROBLASTOMAS='neuroblastomas',
+  OSTEOSARCOMA='osteosarcoma',
+  OTHERCANCERS='otherCancers',
+}
+enum HospitalsDropdownValues {
+  BOSHOSPITAL='bostonChildrenHospital',
+  CTHOPSITAL='connecticutChildrenHospital',
+  DANAHOSPITAL='danaFarberHospital',
+  HASBROHOSPITAL='hasbroChildrenHospital',
+  HOPEHOSPITAL='hopeHealthHospital',
+  MASSGENHOSPITAL='massGeneralHospital',
+  TUFTSHOSPITAL='tuftsMedicalHospital',
+  UMASSHOSPITAL='umassHospital',
+  VERMONTHOSPITAL='universityOfVermontHospital',
+  OTHERHOSPITALS='otherHospitals',
+}
+interface MedicalFormValues {
+  // childDiagnosis: string;
+  diagnosisDate: Date;
+  childPhysician: string;
+  // hospitalName: string;
+  hospitalAddress: string;
+  hospitalCity: string;
+  hospitalState: string;
+  hospitalZipcode: string;
+  hospitalPhone: string;
+  doctorsExplanation: string;
+  doctorName: string;
+  doctorTitle: string;
+  doctorSignature: string;
+  doctorSignatureDate: Date;
+  socialWorkerEmail: string;
+  medicalFormNotes: string;
+}
+
+interface ExampleFormProps {
+  onSubmit?: (values: Partial<MedicalFormValues>) => Promise<void>;
+}
+
+const alertOnSubmit = async (values: FormValues) => {
+  // import and call method on api
+  await new Promise((resolve) => setTimeout(resolve, 1000));
+  alert(JSON.stringify(values, null, 2));
+};
+
+const ExampleForm: React.FC<ExampleFormProps> = ({ onSubmit }) => (
+  <Form onSubmit={onSubmit ?? alertOnSubmit} initialValues={{}}>
+
+    <Select placeholder="Select Child's Diagnosis" size='md'>
+      <option value= {CancersDropdownValues.BRAINCANCERS}> Brain Cancers </option>
+      <option value={CancersDropdownValues.LEUKEMIAS}> Leukemias </option>
+      <option value={CancersDropdownValues.LYMPHOMAS}> Lymphomas </option>
+      <option value={CancersDropdownValues.THYROIDCANCER}> Thyroid Cancer </option>
+      <option value={CancersDropdownValues.GERMCELLTUMORS}> Germ Cell & Gonadal Tumors </option>
+      <option value={CancersDropdownValues.NEUROBLASTOMAS}> Neuroblastomas </option>
+      <option value={CancersDropdownValues.OSTEOSARCOMA}> Osteosarcoma </option>
+      <option value={CancersDropdownValues.OTHERCANCERS}> Other </option>
+    </Select>
+    <Field name="childPhysician">
+      {({ field, form }: FieldProps) => (
+        <FormControl
+          isInvalid={Boolean(
+            form.errors['childPhysician'] &&
+              form.touched['childPhysician']
+          )}
+        >
+          <FormLabel htmlFor={'childPhysician'}>Child's Physician</FormLabel>
+          <Input {...field} id={'childPhysician'} />
+        </FormControl>
+      )}
+    </Field>
+
+    <Select placeholder="Hospital Name" size='md'>
```

Comment:
i think this should probably be wrapped in a field/form control like all the other fields and be required!  also, if you could add an optional text entry field below this dropdown that says something like "If Other, please specify" that would be great